tools/syntax/VsCode/syntaxes/bond.tmLanguage

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>fileTypes</key>
+    <array>
+        <string>bond</string>
+    </array>
+    <key>name</key>
+    <string>Bond</string>
+    <key>patterns</key>
+    <array>
+        <dict>
+            <key>include</key>
+            <string>#commonStuff</string>
+        </dict>
+        <dict>
+            <key>captures</key>
+            <dict>
+                <key>1</key>
+                <dict>
+                    <key>name</key>
+                    <string>keyword.other.namespace.source.bond</string>
+                </dict>
+                <key>2</key>
+                <dict>
+                    <key>name</key>
+                    <string>storage.type.bond</string>
```

Comment:
'constant.language.language.bond' instead of 'storage.type.bond'?

I'm using the [TextMate Language Grammars](http://manual.macromates.com/en/language_grammars) page as my reference. Happy to be pointed to more up-to-date/VS Code-specific references if they apply.